lib/Common/Memory/PageAllocator.cpp

```diff
@@ -299,6 +301,11 @@ PageSegmentBase<T>::IsAllocationPageAligned(__in char* address, size_t pageCount
         return true;
     }
 
+    if (nextIndex != nullptr)
+    {
+        *nextIndex = (uint) ((reinterpret_cast<uintptr_t>(address) % (mask + 1)) / AutoSystemInfo::PageSize);
```

Comment:
Actually, probably not- nextIndex is intended to be the index of the next *aligned* page, yes? That makes sense